app/src/components/GameHeaderPart.vue

```diff
@@ -9,6 +10,14 @@ import userProfileStore, {
   loadUserProfile,
   resetUserProfile,
 } from '../store/userProfile'
+import { i18next } from '../i18n' // Import your i18n instance
+import { ref } from 'vue'
+
+const currentLanguage = ref('en')
+function changeLanguage(lang) {
+  i18next.changeLanguage(lang)
+  currentLanguage.value = lang
+}
```

Comment:
I think it would be better to store in global state, but we can revise them in the ongoing Pinia refactoring.